SU2_CFD/src/solvers/CNEMOEulerSolver.cpp

```diff
@@ -2151,212 +2151,146 @@ void CNEMOEulerSolver::BC_Outlet(CGeometry *geometry, CSolver **solver_container
-//
-//      /*--- Set various quantities in the solver class ---*/
-//      conv_numerics->SetNormal(Normal);
-//      conv_numerics->SetConservative(U_domain, U_inlet);
-//      conv_numerics->SetPrimitive(V_domain, V_inlet);
-//
-//      /*--- Pass supplementary info to CNumerics ---*/
-//      conv_numerics->SetdPdU(nodes->GetdPdU(iPoint), node_infty->GetdPdU(0));
-//      conv_numerics->SetdTdU(nodes->GetdTdU(iPoint), node_infty->GetdTdU(0));
-//      conv_numerics->SetdTvedU(nodes->GetdTvedU(iPoint), node_infty->GetdTvedU(0));
-//      conv_numerics->SetEve(nodes->GetEve(iPoint), node_infty->GetEve(0));
-//      conv_numerics->SetCvve(nodes->GetCvve(iPoint), node_infty->GetCvve(0));
-//
-//      if (dynamic_grid)
-//        conv_numerics->SetGridVel(geometry->nodes->GetGridVel(iPoint),
-//                                  geometry->nodes->GetGridVel(iPoint));
-//
-//      /*--- Compute the residual using an upwind scheme ---*/
-//      auto residual = conv_numerics->ComputeResidual(config);
-//      LinSysRes.AddBlock(iPoint, residual);
-//
-//      /*--- Jacobian contribution for implicit integration ---*/
-//      //if (implicit)
-//      //  Jacobian.AddBlock2Diag(iPoint, residual.jacobian_i);
-//
-//      /*--- Viscous contribution ---*/
-//      if (viscous) {
-//
-//        /*--- Set the normal vector and the coordinates ---*/
-//        visc_numerics->SetNormal(Normal);
-//        su2double Coord_Reflected[MAXNDIM];
-//        GeometryToolbox::PointPointReflect(nDim, geometry->nodes->GetCoord(Point_Normal),
-//                                           geometry->nodes->GetCoord(iPoint), Coord_Reflected);
-//        visc_numerics->SetCoord(geometry->nodes->GetCoord(iPoint), Coord_Reflected);
-//
-//        /*--- Primitive variables, and gradient ---*/
-//        visc_numerics->SetPrimitive(V_domain, V_inlet);
-//        visc_numerics->SetPrimVarGradient(nodes->GetGradient_Primitive(iPoint), nodes->GetGradient_Primitive(iPoint));
-//
-//        /*--- Laminar viscosity ---*/
-//        visc_numerics->SetLaminarViscosity(nodes->GetLaminarViscosity(iPoint), nodes->GetLaminarViscosity(iPoint));
-//
-//        /*--- Compute and update residual ---*/
-//        auto residual = visc_numerics->ComputeResidual(config);
-//        LinSysRes.SubtractBlock(iPoint, residual);
-//
-//        /*--- Jacobian contribution for implicit integration ---*/
-//        //if (implicit)
-//        //  Jacobian.SubtractBlock2Diag(iPoint, residual.Jacobian_i);
-//      }
-//
-//    }
-//  }
-//
-//  /*--- Free locally allocated memory ---*/
-//  delete [] U_domain;
-//  delete [] U_inlet;
-//  delete [] V_domain;
-//  delete [] V_inlet;
-//  delete [] Normal;
+ unsigned short iDim;
+ unsigned long iVertex, iPoint, Point_Normal;
+
+ bool implicit = (config->GetKind_TimeIntScheme() == EULER_IMPLICIT);
+ bool dynamic_grid  = config->GetGrid_Movement();
+ bool viscous       = config->GetViscous();
+ string Marker_Tag = config->GetMarker_All_TagBound(val_marker);
```

Comment:
Declare things as needed not at the top, const everything
```suggestion
 const bool implicit = (config->GetKind_TimeIntScheme() == EULER_IMPLICIT);
 const bool dynamic_grid = config->GetGrid_Movement();
 const bool viscous = config->GetViscous();
 const auto Marker_Tag = config->GetMarker_All_TagBound(val_marker);
```